iot/iot-service-base/pom.xml

```diff
@@ -46,6 +46,24 @@
       <version>${spring.boot.version}</version>
     </dependency>
 
+    <dependency>
+      <groupId>org.infinispan</groupId>
+      <artifactId>infinispan-client-hotrod</artifactId>
+      <optional>true</optional>
+    </dependency>
```

Comment:
I am curios, does this module really have a dependency on Infinispan?

Reply:
looks like no : 
```
+- org.infinispan:infinispan-client-hotrod:jar:9.4.15.Final:compile
[INFO] |  +- org.infinispan:infinispan-commons:jar:9.4.15.Final:compile
[INFO] |  |  \- org.jboss.logging:jboss-logging:jar:3.3.0.Final:compile
[INFO] |  +- org.jboss.marshalling:jboss-marshalling-osgi:jar:2.0.6.Final:compile
[INFO] |  \- org.jboss.spec.javax.transaction:jboss-transaction-api_1.2_spec:jar:1.1.1.Final:compile
```